nflow-engine/src/main/resources/scripts/db/update-5.5.0-x/db2.update.ddl.sql

```diff
@@ -0,0 +1,4 @@
+alter table nflow_workflow add started timestamp(3);
```

Comment:
xxx.create.ddl.sql files also add `started` field to nflow_archive_workflow table but these xxx.update.ddl.sql files do not add `started` field

Reply:
Also should we have UPDATE to fill started fields also for nflow_archive_workflow here?

Reply:
Yes, fixed